src/Middleware/OutputCaching/samples/OutputCachingSample/Startup.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Microsoft.AspNetCore.OutputCaching;
+using Microsoft.AspNetCore.OutputCaching.Policies;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOutputCaching(options =>
+{
+    options.Policies.Add(new EnableCachingPolicy());
+
+    options.Profiles["NoCache"] = new OutputCachePolicyBuilder().NoStore().Build();
+
+    // Tag any request to "/blog**
+    options.Policies.Add(new OutputCachePolicyBuilder().Path("/blog").Tag("blog").Build());
+
+});
+
+var app = builder.Build();
+
+app.UseOutputCaching();
+
+app.MapGet("/", Gravatar.WriteGravatar);
+
+app.MapGet("/cached", Gravatar.WriteGravatar).OutputCache();
```

Comment:
`OutputCache` -> `CacheOutput`.

We like using a verb for method names. With other `IEnpointConventionBuilder` methods we've used `With` (e.g. WithName, WithOpenApiDescription), `Require` (`RequireAuthorization`, `RequireCors`), `Produces/Accepts`, and `Add` (`AddFilters`).

We think a present tense "Cache" as a verb here makes sense.